Latest version cannot be installed

After the latest version update I get the following error when installing it via `npm i Haraka`.
```
npm WARN enoent ENOENT: no such file or directory, open '/Users/christianvaas/Desktop/tst/package.json'
npm WARN tst No description
npm WARN tst No repository field.
npm WARN tst No README data
npm WARN tst No license field.

npm ERR! path /Users/me/Desktop/tst/node_modules/Haraka/bin/haraka
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/me/Desktop/tst/node_modules/Haraka/bin/haraka'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/me/.npm/_logs/2018-11-18T10_07_41_313Z-debug.log
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,7 +1,6 @@
 node_modules
 !tests/node_modules
 .coveralls.yml
-Haraka
 *~
 bower_components
 coverage

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 node_modules
 !tests/node_modules
 .coveralls.yml
+Haraka-*.gz
 *~
 bower_components
 coverage

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,8 +6,9 @@ node_js:
 #   - 0.12      # maint. ended 2016-12-31
 #   - 4         # maint. ended 2018-04
     - "6"       # maint. ends  2019-04
-    - "8"       # LTS
-    - "10"      # current
+    - "8"       # maint. ends  2020-01
+    - "10"      # active until 2020-04
+    - "11"      # current
 
 matrix:
   fast_finish: true
@@ -33,4 +34,3 @@ sudo: false
 cache:
     directories:
         - node_modules
-

--- a/Changes.md
+++ b/Changes.md
@@ -5,6 +5,13 @@
 ### Fixes
 
 
+## 2.8.23 - Nov 18, 2018
+
+### Changes
+
+* tighten Haraka pattern in .gitignore #2542
+
+
 ## 2.8.22 - Nov 17, 2018
 
 ### New Features

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "license": "MIT",
   "description": "An SMTP Server project.",
   "keywords": [ "haraka", "smtp", "server", "email" ],
-  "version": "2.8.22",
+  "version": "2.8.23",
   "homepage": "http://haraka.github.io",
   "repository": {
     "type": "git",

